store/manager.go

```diff
@@ -24,45 +27,53 @@ type Manager struct {
 
 	s3 common.PrecomputedKeyStore // for op keccak256 commitment
 	// For op generic commitments & standard commitments
-	eigenda   common.GeneratedKeyStore // v0 da commitment version
-	eigendaV2 common.GeneratedKeyStore // v1 da commitment version
-	writeV2   bool                     // write blobs to EigenDAV2 backend
+	eigenda      common.GeneratedKeyStore // v0 da commitment version
+	eigendaV2    common.GeneratedKeyStore // v1 da commitment version
+	disperseToV2 *atomic.Bool             // write blobs to EigenDAV2 backend
```

Comment:
any reason to take the atomic here? feels like taking the bool and wrapping to atomic.Bool inside the constructor would be nicer interface? unless the atomic is used for something else beforehand already?